Tests/JWEHeaderTests.swift

```diff
@@ -259,6 +261,9 @@ class JWEHeaderTests: XCTestCase {
 
         XCTAssertEqual(header.parameters["crit"] as? [String], crit)
         XCTAssertEqual(header.crit, crit)
+
+        XCTAssertEqual(header.parameters["zip"] as? String, zip)
+        XCTAssertEqual(header.zip, zip)
     }
 
```

Comment:
Could you also add a tiny test that checks the computed `compressionAlgorithm` property of the header? Just for completeness.

Reply:
Yep, thanks for noticing. Will add them.